extention/app.js

```
@@ -0,0 +1,470 @@
+  }
+}
+
+function throttle(func, limit = 100) {
+  createStickyDiv();
+  createFloatingMenu();
+  let inThrottle;
+  let lastResult;
+  domSelected = false;
+  return function () {
+    const args = arguments;
+    const context = this;
```

Biome (1.9.4)

[error] This aliasing of this is unnecessary.

Arrow functions inherits `this` from their enclosing scope.
Safe fix: Use this instead of an alias.


(lint/complexity/noUselessThisAlias)